charts/redpanda/templates/secrets.yaml

```diff
@@ -62,33 +65,33 @@ stringData:
           status=$(${CURL_MAINTENANCE_DELETE_CMD})
           sleep 0.5
       done
-    }
 
-    export -f postStartHook
-    timeout {{ div .Values.statefulset.terminationGracePeriodSeconds 2 }} bash -c "postStartHook"
-    true
+{{- if and .Values.auth.sasl.enabled (not (empty .Values.auth.sasl.secretRef )) }}
+      # Setup and export SASL bootstrap-user
+      IFS=":" read -r USER_NAME PASSWORD MECHANISM < $(find /etc/secrets/users/* -print)
+      MECHANISM=${MECHANISM:-{{- include "sasl-mechanism" . }}}
+      rpk acl user create ${USER_NAME} --password=${PASSWORD} --mechanism ${MECHANISM} {{ template "rpk-flags-no-brokers-no-sasl" $ }} || true
```

Comment:
Why user needs to be created in post start hook?

Reply:
I don't know. Check git history, I guess.